viewer/viewer.js

```diff
@@ -5474,6 +5474,7 @@ function localSessionDetailReturnFull(req, res, session, incoming) {
   if (req.packetsOnly) { // only return packets
     res.render('sessionPackets.pug', {
       filename: 'sessionPackets',
+      cache: true,
```

Comment:
please remove this file from PR

Reply:
Did you want the whole file removed or just this line? Do you want this is a separate PR? [I found that it made a big difference in performance]

Reply:
In general 1 thing per PR, although letting the db.c slide.  Also in general we like to leave caching off because there are many issues with new installs and not restarting etc that the slight performance hit fixes.  Finally setting NODE_ENV=production many times turns on this type of caching, although I don't know for certain in this case.  So yes a separate PR, probably using NODE_ENV=production if it doesn't use that already would be the right way to go.